rcl/include/rcl/subscription.h

```diff
@@ -104,7 +104,7 @@ rcl_get_zero_initialized_subscription(void);
  * ```
  *
  * The rosidl_message_type_support_t object contains message type specific
- * information used to publish messages.
+ * information used to subscribe messages.
```

Comment:
```suggestion
 * information used to subscribe to messages.
```

Reply:
Thank you. Updated in https://github.com/ros2/rcl/pull/1073/commits/8d499463e64fb07507d431e0c7112934d5d33082